javascript-methods/main.js

```diff
@@ -0,0 +1,42 @@
+var numbers = [25, 47, 69];
+var maximumValue = Math.max(25, 47, 69);
+console.log('numbers: ', numbers);
+console.log('MaximumValue: ', maximumValue);
+
+var heroes = ['Superman', 'Arnold Schwarzenegger', 'Hulk', 'Batman'];
+var randomNumber = Math.random();
+randomNumber = heroes.length * randomNumber;
+var randomIndex = Math.floor(randomNumber);
+console.log('randomIndex: ', randomIndex);
+var randomHero = heroes[randomIndex];
+console.log('randomHero: ', randomHero);
+
+var library = [
+  {
+    title: 'Brave New World',
+    author: 'Aldous Huxley'
+  }
+];
+var lastBook = library.pop();
+console.log('lastBook: ', lastBook);
+
+var js = {
+  title: 'JavaScript for Impatient Programmers',
+  author: 'Dr. Axel Rauschmayer'
+};
+var css = {
+  title: 'CSS Secrets',
+  author: 'Lea Verou'
+};
+library.push(js);
+library.unshift(css);
+library.slice(1);
```

Comment:
This won't behave how you expect it to, please read the docs on the splice method.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice
![Uploading image.png…]()

Reply:
![image](https://user-images.githubusercontent.com/52118958/102961227-a17d6700-4498-11eb-9935-af95aa667ab7.png)